packages/lesswrong/server/search/elastic/ElasticConfig.ts

```diff
@@ -390,7 +390,6 @@ const elasticSearchConfig: Record<SearchIndexCollectionName, IndexConfig> = {
         scoring: {type: "numeric", pivot: 10},
       },
     ],
-    karmaField: "baseScore",
```

Comment:
Was this intentionally removed in this PR?

Reply:
Yes, Jim had added it in a different branch but it's unnecessary because it by default uses baseScore as the karma field